website/content/docs/configuration/session-recording/create-storage-bucket.mdx

```diff
@@ -314,6 +315,84 @@ Complete the following steps to create a storage bucket in Boundary.
 </Tab>
 </Tabs>
 
+</Tab>
+
+<Tab heading="S3-compliant">
+
+Complete the following steps to create a storage bucket in Boundary.
```

Comment:
not sure if this is the right place for this, but we probably want to call out that we're using hitachi as the example 

```suggestion
Complete the following steps to create a storage bucket in Boundary using an S3-compliant storage provider. Hitachi Content Platform is used as an example below.
```